src/Resque/Redis/RedisWorkerRegistry.php

```diff
@@ -85,9 +90,9 @@ public function deregister(WorkerInterface $worker)
     {
         $id = $worker->getId();
 
-        $worker->halt();
```

Comment:
Was this causing issues if deregistering from a different host, or a dead worker?... or both?